### PR TITLE
feature(pu): unizero efficiency optimization and ddp configs

### PR DESCRIPTION
**Pull request by puyuan1996**

### CODE REVIEW

File: lzero/model/unizero_model.py [hunk 1]
```diff
@@ -98,24 +98,26 @@ def __init__(
...
             )
-            # TODO: we should change the output_shape to the real observation shape
-            self.decoder_network = LatentDecoder(embedding_dim=world_model_cfg.embed_dim, output_shape=(3, 64, 64))
 
             # ====== for analysis ======
             if world_model_cfg.analysis_sim_norm:
                 self.encoder_hook = FeatureAndGradientHook()
                 self.encoder_hook.setup_hooks(self.representation_network)
+            # TODO: we should change the output_shape to the real observation shape
+            # self.decoder_network = LatentDecoder(embedding_dim=world_model_cfg.embed_dim, output_shape=(3, 64, 64))
```
Comment (PaParaZz1) [inline lzero/model/unizero_model.py:107]: add comments about the reason why comment this line

File: lzero/model/unizero_world_models/lpips.py [hunk 2]
```diff
@@ -21,16 +21,16 @@ def __init__(self, use_dropout: bool = True):
         self.scaling_layer = ScalingLayer()
         self.chns = [64, 128, 256, 512, 512]  # vg16 features
         # Comment out the following line if you don't need perceptual loss
-        self.net = vgg16(pretrained=True, requires_grad=False)
+        # self.net = vgg16(pretrained=True, requires_grad=False)
```
Comment (PaParaZz1) [inline lzero/model/unizero_world_models/lpips.py:24]: why comment this line

File: lzero/worker/muzero_evaluator.py [hunk 3]
```diff
@@ -449,6 +449,8 @@ def eval(
 
         if get_world_size() > 1:
             objects = [stop_flag, episode_info]
+            print(f'objects: {objects}')
```
Comment (PaParaZz1) [inline lzero/worker/muzero_evaluator.py:452]: remove commented code

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

**Comment by puyuan1996** [comment]
We will have a polised ddp PR.

**Pull request closed without merging**